scripts/helm_redeploy_all_users.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env sh
+
+ENVIRONMENT=""
+HELM_CHART="config-user"
+HELM_CHART_VERSION="0.2.10"
+
+while true; do 
+	read -p "Enter the environment, i.e. dev or alpha, and press [Enter]: " environment
+   	case $environment in
+		[dev]*	) ENVIRONMENT=dev; break;;
+		[alpha]*) ENVIRONMENT=alpha; break;;
+		*	) echo "Only dev or alpha are acceptable answers"; exit;;
+	esac
+done
+
+CHART="../../analytics-platform-helm-charts/charts/${HELM_CHART}"
+RSTUDIO_CHART="../../analytics-platform-helm-charts/charts/rstudio"
+JUPYTER_CHART="../../analytics-platform-helm-charts/charts/jupyter-lab"
+RSTUDIO_CONFIG_FILE="../../analytics-platform-config/chart-env-config/${ENVIRONMENT}/rstudio.yml"
+JUPYTER_CONFIG_FILE="../../analytics-platform-config/chart-env-config/${ENVIRONMENT}/jupyter.yml"
+# USERNAME=$(helm list --max 10000 | grep "${HELM_CHART}" | awk '{print $1}' | sed "s/${HELM_CHART}-//g")
```

Comment:
The script is called "redeploy_all_users" so I think you should uncomment the next line instead of this one, for the version you commit.

Reply:
I'll change the name then to say per user

Reply:
We'll leave this all users and we need to roll this out to list of users once we finish testing with the beta users